doc/src/user_guide.rst

```diff
@@ -875,19 +875,27 @@ file with
 
         - ``rho`` --- density
 
-           *float*
+           *constant float or a function of physical location 'x', 'y', 'z' 
```

Comment:
For consistency with elsewhere in the User Guide, I would just put *float* | *string* for the moment. Same comment applies in eight places below as well.